docs/install.md

```diff
@@ -44,7 +44,7 @@ You can [build Redis Desktop Manager from source](install.md#build-from-source).
 2. Get source code:
  
 ```bash
-git clone --recursive https://github.com/uglide/RedisDesktopManager.git -b 0.8.0 rdm && cd ./rdm
+git clone --recursive https://github.com/uglide/RedisDesktopManager.git -b 0.8.8 rdm && cd ./rdm
```

Comment:
This change introduces unwanted behaviour and should be reverted
Old one: Clone **branch** 0.8.0
New one: Clone **tag** 0.8.8 

It's bad to hardcode specific tags because they became outdated very fast.